### DISCUSSION TITLE
Highlight or call out variables not in use in a Measure

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1037] [Ideas]
**Is your feature request related to a problem? Please describe.**
When I am modifying or debugging measures, I often create variables or change which variables are used to get the result. I often wind up with variables that are unnecessary in the final result.

**Describe the solution you'd like**
I'd like some sort of visual indicator in the Expression editor for variables that are not called anywhere else in the expression. I know it doesn't cause a performance issue, just a code readability issue, and it confuses Future Ed sometimes.

It could be a button to highlight unused variables, or even variable chains. Example:

varFirstValue is used by varSecondValue, and that is used in varThirdValue, but none of those can actually impact the results of the measure, they were orphaned debug code or left over from the original version of a measure that got reworked.

**Describe alternatives you've considered**
I currently just comment out suspect variables and see if I get error messages.

**Comment by corycundy** [comment]
I think this is a great idea!

**Comment by DBojsen** [comment]
Great idea - I put it on the backlog.  No ETA yet.

**Comment by mlonsk** [comment]
Seems to be same suggestion as #1055

**Comment by otykier** [comment]
Now available in [3.12.0](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.12.0)! Thanks for the suggestion!
